### PR TITLE
chore: move rviz plugins from common to visualization/ folder

### PR DESCRIPTION
**Pull request by esteve**
## Description

Move `rviz` plugins from comon to its own `visualization/` folder so that creating Docker images for only `common` is easier.

## Related links

**Parent Issue:**

- https://github.com/autowarefoundation/autoware/issues/5076
- https://github.com/autowarefoundation/autoware/issues/5077

## Notes for reviewers

None.

## Interface changes

None.

## Effects on system behavior

None.

### CODE REVIEW

**Comment by esteve** [comment]
@xmfcx this simply moves the rviz plugins to the visualization folder, there are no code changes. After this PR gets merged, I'll trigger an update of CODEOWNERS file via the corresponding GH action.

**Review by xmfcx — APPROVED** [review]
Only moves packages :+1:

**Pull request merged**
